src/client/v2/algod/algod.ts

```diff
@@ -25,6 +25,8 @@ import {
   AlgodTokenHeader,
   CustomTokenHeader,
 } from '../../urlTokenBaseHTTPClient';
+import LightBlockHeaderProof from './lightBlockHeaderProof';
+import StateProof from './stateproof';
```

Comment:
not sure about the conventions of this repo, but should these single imports be inserted alphabetically above?

Reply:
They do look to be sorted somewhat, so I updated that.